tests/dtm/test_multi_m0d_process_restart.py

```diff
@@ -0,0 +1,208 @@
+        for each in resp:
+            self.log.info("Editing Deployment for %s", each)
+            resp = self.dtm_obj.edit_deployments_for_delay(self.master_node_list[0], each, M0D_SVC)
+            assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Edit deployment done for all data pods")
+        self.log.info("Sleep of %s secs", self.test_cfg['edit_deployment_delay'])
+        time.sleep(self.test_cfg['edit_deployment_delay'])
+        self.log.info("Check the overall status of the cluster.")
+        resp = self.ha_obj.check_cluster_status(self.master_node_list[0])
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Cluster status is online.")
+        self.log.info("Get the value of K for the given cluster.")
+        resp = self.ha_obj.get_config_value(self.master_node_list[0])
+        if resp[0]:
+            self.kvalue = int(resp[1]['cluster']['storage_set'][0]['durability']['sns']['parity'])
+        else:
+            self.log.info("Failed to get parity value, will use 1.")
+            self.kvalue = 1
+        self.log.info("The cluster has %s parity pods", self.kvalue)
+
+    def teardown_method(self):
+        """Teardown class method."""
+        if not self.test_completed:
+            self.log.info("Test Failure observed, collecting support bundle")
+            path = os.path.join(LOG_DIR, LATEST_LOG_FOLDER)
+            resp = support_bundle_utils.collect_support_bundle_k8s(
+                local_dir_path=path, scripts_path=const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp)
+        if self.iam_user:
+            self.log.info("Cleanup: Deleting objects created during test.")
+            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.iam_user, True)
+            assert_utils.assert_true(resp[0], resp[1])
+        if os.path.exists(self.test_dir_path):
+            system_utils.remove_dirs(self.test_dir_path)
+        # TODO : Redeploy setup after test completion.
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    def test_write_during_multi_m0d_restart(self):
+        """Verify write during multiple m0d restart using pkill."""
+        self.log.info("STARTED: Verify write during multiple m0d restart using pkill")
+        log_file_prefix = 'test-XXXXX'
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Create bucket for IO operations")
+        self.s3_test_obj.create_bucket(self.bucket_name)
+
+        self.log.info("Step 2: Start write Operations in background:")
+        proc_write_op = multiprocessing.Process(target=self.dtm_obj.perform_write_op,
+                                                args=(self.bucket_name, self.object_name,
+                                                      self.test_cfg['clients'],
+                                                      self.test_cfg['samples'], log_file_prefix,
+                                                      que, self.test_cfg['size'], 1,
```

Comment:
Loop count needs to be increased as write for 256Mb with one loop won't last for the duration of multiple process restarts.

Reply:
currently test case does not have any size related information, hence kept it default.